app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    file_name = command.split()
+    if len(file_name) == 3:
+        if file_name[0] == "cp" and file_name[1] != file_name[2]:
```

Comment:
You could perform unpacking at the beginning of your code after the `split()` operation so that you would avoid indexing. It could make code more descriptive.